pkgs/servers/teleport/default.nix

```diff
@@ -0,0 +1,50 @@
+# This file was generated by https://github.com/kamilchm/go2nix v2.0-dev
+{ stdenv, buildGoPackage, zip, fetchurl, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "teleport-unstable-${version}";
+  version = "2.4.0";
+
+  goPackagePath = "github.com/gravitational/teleport";
+  subPackages = [ "tool/tctl" "tool/teleport" "tool/tsh" ];
+  buildInputs = [ zip ];
+  postBuild = ''
+    pushd .
+    cd $NIX_BUILD_TOP/go/src/github.com/gravitational/teleport
+    mkdir -p build
+    echo "making webassets"
+    make build/webassets.zip
+    cat build/webassets.zip >> $NIX_BUILD_TOP/go/bin/teleport
+    rm -fr build/webassets.zip
+    cd $NIX_BUILD_TOP/go/bin
+    zip -q -A teleport
+    popd
+    '';
+
+  dontStrip = true;
+
+  # This repo has a private submodule "e" which fetchgit cannot handle
```

Comment:
It seems the commit you were using (`df6caeae7d90e520478244d583cd9e60d924fb32`) is not the commit ID for the tag 2.4.0. So, maybe there is a diff between the deps you genreated and the soruce code you are using.
- You can fetch an archive for a particular commit ID on github or
- Set this attribute: https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchgit/default.nix#L16 or
- Generate deps by checking out tag 2.4.0

Reply:
Correct, that was master a few days ago. Good catch. But we're not longer using deps... so meh.